### ISSUE TITLE
Feature Idea: badges for kanban/gallery cards

### ISSUE THREAD

**Issue by kamre** [mattermost/focalboard#1754]
## Summary
Cards can contain different types of content inside them. It will be useful to know what is inside the card on kabban board or in the gallery without opening the dialog for it.

This can be done with help of additional information that will be shown at the bottom of the card. So called "badges" that give quick summary about what is inside. Here how it can look:
![CardBadgesPOC](https://user-images.githubusercontent.com/5587620/140290285-e94d46eb-9d50-4649-8975-b6460650c507.gif)
Proper implementation should also count checkboxes in the markdown. Can also have a badge for images for example. And probably showing of these badges should be optional because they take place.

## How important this is to me and why
Importance: Medium, will allow better management of the cards.

## Additional context/similar features
Trello has badges for the cards:
![TrelloCardBadges](https://user-images.githubusercontent.com/5587620/140289847-2ae6b966-0695-4652-a5b3-9ed64c554c35.gif)

@chenilim @asaadmahmood what do think about such feature?

**Comment by chenilim** [comment]
Thanks @kamre, and sorry for the delayed response. Yes! This looks great! We are also planning on adding a ["following" feature](https://community-daily.mattermost.com/plugins/focalboard/workspace/qgsck6cts3fwpqwyjiupjm5cde/shared/47aa9bb4-6967-4a96-83c7-11bd6b20f1eb/7e38eab4-890a-4fc3-8f48-a1f9bd1ab822/e49c9a17-7aa9-4cf0-8e55-da7a0b924690?r=762db9c6-305c-4d13-8ce6-9f11e5232c24) in v0.11, so we could have that badge too.

The limitation is the amount of space available - these would take an extra row, so need to be balanced with showing more / other properties. Showing / hiding them could be a toggle in the properties menu.

What do you think @asaadmahmood? cc @wuwinson.

**Comment by kamre** [comment]
Hi @asaadmahmood Can you provide your opinion about this feature?

I have initial implementation and would like to continue working on this feature. So any input on the UI design will be helpful. May be there are better icons for these badges in the Compass icon set that is currently used.

**Comment by asaadmahmood** [comment]
@kamre Are these configurable? so only to be shown when the user selects them?

**Comment by kamre** [comment]
Yes, we need to make this configurable so that user can hide and show badges. This information should be saved in the view settings.

But where to place these hide/show actions? My idea was to use three dots menu:
![image](https://user-images.githubusercontent.com/5587620/145668994-30c6bdba-33b6-40b1-975c-b5206a0845dd.png)
with adding "Hide card badges" or "Show card badges" actions there. But @chenilim proposed also to use `Properties` menu for these actions.
